doc.go

```diff
@@ -0,0 +1 @@
+package go_option
```

Comment:
It looks it will be better to use the name. The approach is used in all tarantoo/go-foo libraries.
```suggestion
package option
```

Reply:
fixed